README.md

```diff
@@ -166,6 +180,20 @@ Delay line.
 	- `DelayTime`
 	- `DelayTimeModulation`
 
+### EQBPFilter
```

Comment:
See naming <= 8 characters in length above.

Reply:
maybe I'll just go`EQBP` - I wanded this one for a specific effect but down the line it could be cool to add the whole BEQ suite for making EQs

Reply:
EQBP is fine. Feel free to add more modules based on the BEQ suite in another PR.